### PR TITLE
Load data from agent and first view

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

File: temboardui/json_encoder.py [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
+    Base class for an object which is serializable to JSON.
+    """
+    def to_json(self):
+        """
+        Serialize the object to JSON
+        Returns:
+            an object which can be encoded by the BaseJSONEncoder.
+        """
+        return dict(((key, val) for key, val in self.__dict__.items()
+                    if not key.startswith("_")))
```
Comment (dlax) [inline temboardui/json_encoder.py:33]: This class does not appear to be used anywhere in the PR. Or maybe I missed something?

Comment (pgiraud) [reply temboardui/json_encoder.py:33]: You have the following statement in the JSONEndocer class (same file):
`if isinstance(obj, JSONizable):`

**Comment by dlax** [comment]
Yes, I noticed that. But it seems that there's no class inheriting from
JSONizable at this point so this code will never apply, if I understand
correctly.

File: temboardui/json_encoder.py (hunk 1 above)
Comment (dlax) [reply temboardui/json_encoder.py:33]: Yes, I noticed that. But it seems that there's no class inheriting from JSONizable at this point so this code will never apply, if I understand correctly.

Comment (pgiraud) [reply temboardui/json_encoder.py]: My bad.

**Pull request merged**